dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
```

Comment:
pathLine is the result of concatenating some nunjucks renders, [source](https://github.com/mozilla/dxr/blob/8e56b088d3bdd074495140cf9ee9711ddf98d3e8/dxr/static_unhashed/js/dxr.js#L110-L116), and path_line.html does not have any |safe usage.